docs/markdown/testing/redbox.md

```diff
@@ -210,18 +210,18 @@ Some interactive example to get the best out of redbox
 
 ## Resources
 
-**[RedboxGroup](/buttongroup)**
+**[RedboxGroup](/web/buttongroup)**
```

Comment:
Should we keep these testing files around? Seems like more work to keep them updated

Reply:
I'd be fine with deleting them. I think @rlingineni wanted to keep them around for reference but that's what git history is for 🙂

I'll put up another PR to remove them

Reply:
here's that PR: https://github.com/pinterest/gestalt/pull/2378

Reply:
Thank you!